build.sbt

```diff
@@ -9,7 +9,7 @@ lazy val project = Project(pluginName, file("."))
     majorVersion := 1,
     makePublicallyAvailableOnBintray := true,
     crossSbtVersions := List("0.13.18", "1.3.4"),
```

Comment:
Do we want 0.13 in here still?

Reply:
left it in for Play 2.5, but now we are removing 2.5 from the plugin so I will remove the SBT version.

Reply:
done